drivers/timer/CMakeLists.txt

```diff
@@ -28,6 +28,7 @@ zephyr_library_sources_ifdef(CONFIG_NPCX_ITIM_TIMER npcx_itim_timer.c)
 zephyr_library_sources_ifdef(CONFIG_NRF_GRTC_TIMER nrf_grtc_timer.c)
 zephyr_library_sources_ifdef(CONFIG_NRF_RTC_TIMER nrf_rtc_timer.c)
 zephyr_library_sources_ifdef(CONFIG_RCAR_CMT_TIMER rcar_cmt_timer.c)
+zephyr_library_sources_ifdef(CONFIG_ULPT_RENESAS_RA ra8_ulpt_timer.c)
```

Comment:
Please change file name to "renesas_ra8_ulpt_timer" to easy detect for Renesas RA maintaining.

Reply:
@thaoluonguw , It's fixed